package.json

```diff
@@ -36,91 +36,73 @@
     "migration:show": "npm run typeorm migration:show"
   },
   "dependencies": {
-    "@jolocom/sdk": "^1.2.0",
-    "@jolocom/sdk-storage-typeorm": "^4.2.0",
-    "@jolocom/vaulted-key-provider": "^0.8.0",
-    "@nestjs/common": "^7.6.15",
-    "@nestjs/config": "^0.5.0",
-    "@nestjs/core": "^7.6.15",
-    "@nestjs/graphql": "^7.10.3",
-    "@nestjs/jwt": "^7.2.0",
-    "@nestjs/passport": "^7.1.5",
-    "@nestjs/platform-express": "^7.6.15",
-    "@nestjs/typeorm": "^7.1.5",
+    "@nestjs/axios": "^0.0.1",
+    "@nestjs/common": "^8.0.5",
+    "@nestjs/config": "^1.0.1",
+    "@nestjs/core": "^8.0.5",
+    "@nestjs/graphql": "^8.0.2",
+    "@nestjs/jwt": "^8.0.0",
+    "@nestjs/passport": "^8.0.0",
+    "@nestjs/platform-express": "^8.0.5",
+    "@nestjs/typeorm": "^8.0.1",
     "@ory/kratos-client": "^0.6.3-alpha.1",
-    "@types/crypto-js": "^4.0.1",
-    "@types/express-session": "^1.17.3",
     "apollo-server-express": "^2.22.2",
-    "axios": "^0.21.1",
-    "cd": "^0.3.3",
-    "class-transformer": "^0.3.2",
-    "class-validator": "^0.12.2",
+    "class-transformer": "^0.4.0",
+    "class-validator": "^0.13.1",
     "cross-env": "^7.0.3",
-    "crypto-js": "^4.0.0",
-    "express-session": "^1.17.1",
-    "graphql": "^15.5.0",
-    "graphql-tools": "^6.2.6",
+    "crypto-js": "^4.1.1",
+    "graphql": "^15.5.1",
     "graphql-type-json": "^0.3.2",
-    "graphql-upload": "^11.0.0",
-    "helmet": "^4.4.1",
-    "image-size": "^0.9.7",
-    "ipfs-http-client": "^49.0.4",
-    "jolocom-lib": "^5.1.0",
-    "js-yaml": "^4.1.0",
-    "jwks-rsa": "^2.0.3",
+    "graphql-upload": "^12.0.0",
+    "helmet": "^4.6.0",
+    "image-size": "^1.0.0",
+    "ipfs-http-client": "^51.0.0",
+    "jwks-rsa": "^2.0.4",
     "matrix-js-sdk": "^9.11.0",
     "module-alias": "^2.2.2",
     "mysql": "^2.18.1",
-    "nest-winston": "^1.4.0",
-    "nodemon": "^2.0.7",
-    "normalizer": "^1.2.7",
+    "nest-winston": "^1.5.0",
+    "nodemon": "^2.0.12",
     "passport": "^0.4.1",
     "passport-custom": "^1.1.1",
     "passport-jwt": "^4.0.0",
     "reflect-metadata": "^0.1.13",
     "replace-special-characters": "^1.2.5",
     "rimraf": "^3.0.2",
-    "rxjs": "^6.6.7",
+    "rxjs": "^7.3.0",
     "sqlite3": "^5.0.2",
-    "typeorm": "0.2.32",
-    "uuid": "^8.3.2",
+    "typeorm": "^0.2.35",
```

Comment:
Interesting if this is now working, would imply that the bug was triggered by one of the dependencies that were removed...

Reply:
Which bug and how can I check? I recall there was a bug but I haven't tested this update against it...

Reply:
https://github.com/alkem-io/server/issues/1088

Reply:
Haven't seen that error running the application, merging the PR